### PR TITLE
Check accessibility

### PR DESCRIPTION
**Pull request by MahdiRezaeiDev**
Portfolio project: fix accessibility problems
in this branch, we have made the following changes.

- Added a clear value for the `alt` attribute in images.
- Update the font size unit to `rem` in order to be resizeable by the user.
- Add the `aria-label` property to input elements to have labels for screen readers.

Also, I have checked the following accessibility  points:

- Page title: no issue found.
- Image text alternatives: [Added a clear value for the `alt` attribute in images](https://github.com/MahdiSohaily/Portfolio/blob/main/index.html#L52-L65)
- Text heading: no issue found.
- Color Contrast: no issue found.
- Resize : [Update the font size unit to `rem` in order to be resizeable by the user](https://github.com/MahdiSohaily/Portfolio/blob/check-accessibility/src/css/style.css#L140)
- Interaction: [Add the `aria-label` property to input elements to have labels for screen readers](https://github.com/MahdiSohaily/Portfolio/blob/check-accessibility/index.html#L312-L314)
- Moving content: no issue found.
- Multi-media: no issue found.
- The basic structure of the page: no issue found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,15 +22,16 @@
...
     <nav class="wrapper padding-inline-300">
-      <div class="cub">
+      <div class="area">
+        <div class="cub"></div>
         <a href="#" class="logo">Mahdi Rezaei</a>
       </div>
       <ul class="main-nav">
         <li><a href="#portfolio">Portfolio</a></li>
         <li><a href="#about">About</a></li>
         <li><a href="#contact">Contact</a></li>
```
Comment (jasonfelice) [inline index.html:32]: - [x] Kindly make sure nav links takes you to their respective section. You can fix this by adding `id` to each section. 😄

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @MahdiSohaily, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Good commit messages✔️
Professional Readme✔️
Great Design✔️
No Linter Errors✔️

## Required Changes ♻️
- [ ] According to project requirements, you are required to add your report to the description. 
Kindly update your PR description to add that report.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (thecodechaser) [inline index.html:32]: - [x] As requested by the previous reviewer, Please make sure nav links takes you to their respective section. You can fix this by adding an `id` to each section.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @MahdiSohaily, and @jasonfelice👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @MahdiSohaily  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**